src/node_report.cc

```diff
@@ -842,13 +871,14 @@ static void PrintStackFrame(std::ostream& out, Isolate* isolate, Local<StackFram
   char buf[64];
 
   // First print the frame index and the instruction address
+  if( pc != nullptr ) {
```

Comment:
When we get the stack frames for the current stack we also call `isolate->GetStackSample` to get the pc addresses. We can't get those for stack frames that come from an `Error` we are passed so we pass `PrintStackFrame` a `nullptr` for the `pc` value. (We already pass `nullptr` if we couldn't get a stack sample for a frame from the current stack.)

This additional check just stops us printing a pc address of `0x0` for every stack frame of the error we are passed and means we can use the same `PrintStackFrame` function for the current stack and the stack from the Error object.